SightLocator/res/layout-land/text_fragment.xml

```diff
@@ -7,6 +7,20 @@
         android:layout_width="fill_parent"
         android:layout_height="wrap_content"
         android:orientation="vertical" >
+        
+         <TextView
+            android:id="@+id/tilko_vi_lvovi"
```

Comment:
ID is better to start with the element class name, i.e.: text_view_tilko_vi_lvovi, 
it will be easier to find later the apropriate element